Svc/Deframer/Deframer.cpp

```diff
@@ -209,7 +209,11 @@ void Deframer ::processBuffer(Fw::Buffer& buffer) {
         const Fw::SerializeStatus status =
             m_inRing.serialize(&bufferData[offset], serSize);
         // If data does not fit, there is a coding error
-        FW_ASSERT(status == Fw::FW_SERIALIZE_OK, status, offset, serSize);
+        FW_ASSERT(
+            status == Fw::FW_SERIALIZE_OK,
+            status,
```

Comment:
Another missing cast to assert arg type.

Reply:
Fixed